tests/foreman/api/test_contentview.py

```diff
@@ -112,6 +112,104 @@ def test_subscribe_system_to_cv(self):
         self.assertEqual(system_attrs['environment']['id'], lifecycle_env.id)
         self.assertEqual(system_attrs['organization_id'], org.id)
 
+    def test_cv_clone_within_same_env(self):
+        """
```

Comment:
The pattern used on robottelo for multi line docstrings is the following:

```python
"""@Test: description.....
second line of description

@Feature: my feature

...

"""
```

Please make sure to follow that.